src/andromede/expression/parsing/antlr/ExprVisitor.py

```diff
@@ -0,0 +1,74 @@
+# Generated from Expr.g4 by ANTLR 4.13.1
```

Comment:
Maybe add the header here ?

Reply:
We cannot claim the copyright on it since it's generated by a tool.